dandi/metadata.py

```diff
@@ -83,3 +88,62 @@ def get_metadata(path):
     meta["nd_types"] = get_neurodata_types(path)
 
     return meta
+
+
+def nwb_to_json(fpath, out_fpath):
+    io = NWBHDF5IO(fpath, 'r')
```

Comment:
- needs a docstring
- could use `@metadata_cache.memoize_path`
- imho we need a function like this which doesn't dump it into a file -- should return the record (dict).  (see below about the name of `dataset_builder_to_json`), thus be RFed into `nwb_to_dict`.  Outside code could easily dump it into json or yaml depending on the use case